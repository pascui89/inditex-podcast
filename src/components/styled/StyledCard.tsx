import {
    Card,
    styled
  } from '@mui/material';

export const StyledCardOverflowAuto = styled(Card)`
  overflow-y: auto;
  max-height: calc(100vh - 300px);
`;