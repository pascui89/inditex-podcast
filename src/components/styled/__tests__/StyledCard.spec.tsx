import React from 'react';
import { render } from '@testing-library/react';
import { StyledCardOverflowAuto } from '../StyledCard';
import '@testing-library/jest-dom/extend-expect';

global.React = React;

describe('StyledCardOverflowAuto', () => {

  it('should render', () => {
    const { container } = render(<StyledCardOverflowAuto/>);
    expect(container).toMatchSnapshot();
  });
});
