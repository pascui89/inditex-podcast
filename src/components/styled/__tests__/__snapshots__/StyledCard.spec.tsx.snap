// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledCardOverflowAuto should render 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-yq514b-MuiPaper-root-MuiCard-root"
  />
</div>
`;
